doc/source/ray-core/objects/serialization.rst

```diff
@@ -23,6 +23,23 @@ Plasma is used to efficiently transfer objects across different processes and di
 
 Each node has its own object store. When data is put into the object store, it does not get automatically broadcasted to other nodes. Data remains local to the writer until requested by another task or actor on another node.
 
+Object Refs
+~~~~~~~~~~~~~~~~~~~
+
+Ray object refs can be serialized using `ray.cloudpickle`. The object ref can then be deserialized and accessed with `ray.get()`. Note that `ray.cloudpickle` must be used; other pickle tools are not guaranteed to work. Additionally, the process that deserializes the `ObjectRef` must be part of the same Ray cluster that serialized it.
+
+When serialized, the `ObjectRef`'s value will remain pinned in Ray's shared memory object store. The object must be explicitly freed by calling `ray._private.internal_api.free(obj_ref)`.
+
+This should be used as a last resort. Passing `ObjectRefs` through Ray task arguments and return values is the recommended approach.
```

Comment:
Can you put this at the beginning of the subsection instead of the middle?

Reply:
Sure, I moved it.